addon/pods/components/frost-list-item/component.js

```diff
@@ -12,18 +12,30 @@ export default Ember.Component.extend({
   isSelected: Ember.computed.reads('model.isSelected'),
 
   onclick: Ember.on('click', function (event) {
-    if (!Ember.ViewUtils.isSimpleClick(event)) {
+    if (!Ember.ViewUtils.isSimpleClick(event) && !event.shiftKey && !event.metaKey && !event.ctrlKey) {
```

Comment:
This should already be covered by isSimpleClick 

See https://github.com/emberjs/ember.js/blob/ed298ced885b2f4b4435fbccb26b24b9a24bb68f/packages/ember-views/lib/system/utils.js#L7